src/app/features/shows/pages/movie-details/movie-details.component.html

```diff
@@ -1 +1,44 @@
-<p>movie-details works!</p>
+<div *ngIf="loading; else yes">
+  <app-movie-details-skeleton></app-movie-details-skeleton>
+</div>
+<ng-template #yes>
+  <div class="heading">
+    <img src="https://image.tmdb.org/t/p/w500/{{ movie.backdropPath }}" alt="backdrop" class="backdrop">
+    <div class="cart-name">
+      <div class="breadcumb mat-body-1">MaileHereko / Movies</div>
+      <p class="mat-headline-3">{{movie.title}}</p>
+    </div>
+  </div>
+  <div class="info-movie">
+    <div class="poster">
+      <img src="https://image.tmdb.org/t/p/w500/{{ movie.posterPath }}" alt="poster">
+    </div>
+    <div class="desc-movie">
+      <p class="mat-headline-4">{{movie.title}}.</p>
+      <p class="mat-subtitle-1">{{movie.overview}}</p>
+      <div class="movie-item-rating">
+        <img src="/assets/image/star.svg" alt="">
+        {{ movie.voteAverage.toFixed(1) }}
+      </div>
+      <div class="type">
+        <p class="mat-subtitle-2">Type</p>
+        <p class="mat-subtitle-1">Movie</p>
+      </div>
+      <div class="release-date">
+        <p class="mat-subtitle-2">Release Date:</p>
+        <p class="mat-subtitle-1">{{ movie.releaseDate }}</p>
+      </div>
+      <div class="run-time">
+        <p class="mat-subtitle-2">Run time</p>
+        <p class="mat-subtitle-1">{{ movie.runtime }} min</p>
+      </div>
+      <div class="genres">
+        <p class="mat-subtitle-2">Genres</p>
+        <ul>
+          <li *ngFor="let genre of movie.genres"><p class="mat-subtitle-1">{{genre.name}}, </p></li>
```

Comment:
```suggestion
          <p class="mat-subtitle-1">{{movie.genres.join(", ")}}</p>
```
dư dấu phẩy ở cuối sao e?